accessibilityTest/AccessibilityOmnibox.ts

```diff
@@ -38,6 +40,67 @@ export const AccessibilityOmnibox = () => {
       done();
     });
 
+    describe('with keyboard navigation', () => {
+      let input: HTMLInputElement;
+
+      const triggerEvent = (key: KEYBOARD) => {
+        const event = new KeyboardEvent('keyup');
+        Object.defineProperty(event, 'keyCode', {
+          get: () => {
+            return key;
+          }
+        });
+        Object.defineProperty(event, 'which', {
+          get: () => {
+            return key;
+          }
+        });
+        input.dispatchEvent(event);
+      };
+
+      const down = () => triggerEvent(KEYBOARD.DOWN_ARROW);
+      const up = () => triggerEvent(KEYBOARD.UP_ARROW);
+
+      beforeEach(async done => {
+        const omniboxElement = getOmniboxElement();
+        getSearchSection().appendChild(omniboxElement.el);
+        await afterQuerySuccess();
+        showSuggestions(omniboxElement);
+        await afterDelay(500);
+        input = omniboxElement.find('input') as HTMLInputElement;
+        input.focus();
+        done();
+      });
+
+      it('should be accessible when navigating using keyboard down arrow once', async done => {
+        down();
+        const axeResults = await axe.run(getRoot());
+        expect(axeResults).toBeAccessible();
+        done();
+      });
+
+      it('should be accessible when navigating using keyboard up arrow once', async done => {
+        up();
+        const axeResults = await axe.run(getRoot());
+        expect(axeResults).toBeAccessible();
+        done();
+      });
+
+      it('should be accessible when navigating using keyboard down arrow multiple time (wrapping around)', async done => {
+        times(10, () => down());
```

Comment:
Hmm, how do we know that pressing down 10 times is enough to wrap around?

Reply:
Default value is 5. I could configure the component to explicitely say 5 though.